Makefile

```diff
@@ -1,10 +1,13 @@
 .PHONY: doc tutorial travis_tests
 
-ci_tests:
+ci_tests: bids/tests/data/bids-examples
```

Comment:
to make sure that the bids-examples are cloned before running tests in CI, but this might not be obvious to people running tests locally. 

Should I document this somewhere?